### ISSUE TITLE
PROJECT IS DEAD - USE METEOR SLINGSHOT PLUGIN INSTEAD

### ISSUE THREAD

**Issue by mackbrowne** [sebakerckhof/ufs-s3#6]
This plugin doesn't currently work.  I'm experiencing the same syntax error that people were getting a year ago.  I'm aware the readme says the project is in active development, but it really isn't.  Please save other developers the time I lost trying this plugin and failing with it.  

Best regards,
Mack.

**Comment by pedrofg** [comment]
Same here.

**Comment by mackbrowne** [comment]
@pedrofg I wrote that over a year ago.  I completely forgot about it.  Anyway I think it's safe to say that this repo is DEAD AND YOU PROBABLY SHOULDN"T USE IT

**Comment by pedrofg** [comment]
Yes. To anyone looking for another solution: https://github.com/CulturalMe/meteor-slingshot/

**Comment by GbalsaC** [comment]
Yes, repo is no longer maintained, however it's helpful to know the syntax issue is occurring from not loading `s3-patch.js` when importing things into newer Meteor Versions.

This is due to the fact that s3-patch.js tries to overwrite AWS.S3's Prototype, which is no longer loaded as of Meteor 1.4 < as it has full Node support.

You can use this to create your own servce, simply modifying object methods based on these two files: 
s3-patch.js
ufs-s3.js

Where `createReadStream` is actually using `s3.getObject(params).createReadStream()`.
Just reacall AWS s3 version has remained the [same since 2006](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html)

**Comment by GbalsaC** [comment]
@pedrofg Meteor Slingshot has not been active for 3 years, it's older still

**Comment by mackbrowne** [comment]
@GbalsaC the difference is that Meteor slingshot works perfectly out of the box.  I'd say unless someone updates this package with what you mentioned above, this project is currently not working in it's current form and is dead.  (for now at least)

**Comment by GbalsaC** [comment]
Ok, great to know actually!
I'll take a look into slingshoot, thanks for the followup! @mackbrowne  :+1:
